app/models/utility/cell_count_spot_checking.rb

```diff
@@ -137,6 +29,24 @@ def first_replicates
         .values
     end
 
+    # Returns the second replicate wells in the plate. It groups the filtered
+    # wells by the ancestor tube barcode and returns the second well in each
+    # group.
+    def second_replicates
```

Comment:
What if there is no second replicate? Or is it fixed here and there is always a second replicate?
For the Seq/Spare tubes the number of replicates depended on ancestor volume available. Low volume, fewer replicates.

Reply:
This code would skip if no replicate it looks like. That ok?

Reply:
Yes, that is OK. I will update the comment to clarify.

From the post that starts with "After our meeting today," at https://sanger-global.slack.com/archives/C05F21NKATD/p1720104757004039?thread_ts=1719846560.109819&cid=C05F21NKATD

"It would be the same story as before but to pick a replicate from **all LRC Blood Vacs** available rather than just 4. **If a Blood Vac only has 1 replicate, no well should be counted**.